dockerizeit/master/permissions.properties

```diff
@@ -0,0 +1,12 @@
+user1 {
+		userId = hudson.security.ACL.ANONYMOUS_USERNAME
+		permissions = [
+			"hudson.model.Computer.Create",
```

Comment:
The commit message should be as usual "Fix #66 : ..." . May be I accidentally did copy/paste the branch name...Documentation will fix.